source/develop/release-management/features/virt/high-performance-vm.html.md

```diff
@@ -93,60 +91,67 @@ Displayed in 'Resource Allocation' side-tab of the VM dialog.
 
 This option can also be set manually by the user for the cluster of that VM  (add/edit Cluster dialog->Optimization side-tab). 
 
-#### **Enable High availability (for the pinned host)**
+#### **Disable High availability (for the pinned host)**
 Displayed in 'High Availability' side-tab of the VM dialog.
 
 Currently High availability is not supported when migration is disabled.
-We plan to support it with an option for re-running the VM on the same host in oVirt 4.2 and expand it to all cluster hosts as part of the solution for supporting migration in oVirt 4.3.
+We may plan to support it with an option for re-running the VM on the same pinned host in oVirt 4.2 and expand the implementation to all cluster hosts as part of the solution for supporting migration in oVirt 4.3.
 
 #### **Disable the Watchdog device**
 Displayed in 'High Availability' side-tab of the VM dialog.
 
+#### **Enable paravirtualized Random Number Generator PCI (virtio-rng) device** 
+Displayed in 'Random Generator' side-tab of the VM dialog.
+
+This option is automatically set using libvirt defaults ("Period duration"=1000 milliseconds) and can be changed by the user before saving.
+
 ### List of manual configuration settings/warnings for High Performance VM
 
 #### **Set the CPU Pinning topology**
 Displayed in 'Resource Allocation' side-tab of the VM dialog.
 
-In case the CPU Pinning topology is not set upon VM saving,  a recommendation/warning will be displayed in a pop-up and the user will be asked to pin the VM to a host (by selecting the "Start running on: Specific Host" in "Host" side-tab) and verify if VM configuration fits the host configuration (number of cores per socket,  threads per core and number of sockets should match).
+In case the CPU Pinning topology is not set upon VM saving, a recommendation/warning will be displayed in a pop-up and the user will be asked to pin the VM to a host (by selecting the "Start running on: Specific Host" in "Host" side-tab) and verify if VM configuration fits the host configuration (number of cores per socket,  threads per core and number of sockets should match).
```

Comment:
Done.

Regarding "Please try to explain the "make sure it fits the host" scenarios. Topology "fit" is different than hugepages "fit""
Is it enough to write that user should verify:
1. guest huge pages size should be supported by the pinned host.
and
2. guest_num_of_huge_pages_of_size_X  <=  host_num_of_huge_pages_of_size_X?